build/includes/release.mk

```diff
@@ -60,6 +60,7 @@ example-image-markdown.%:
 # Deploys the site by taking in the base version and deploying the previous version
 release-deploy-site: $(ensure-build-image)
 release-deploy-site: DOCKER_RUN_ARGS += -e GOFLAGS="-mod=mod" --entrypoint=/usr/local/go/bin/go
+release-deploy-site: export ENV=HUGO_ENV=snapshot
```

Comment:
```suggestion
```

Rather than doing this, make

```
$(MAKE) site-deploy SERVICE=$$version
```

into

```
$(MAKE) ENV=HUGO_ENV=snapshot site-deploy SERVICE=$$version
```

I expect it will work perfectly, and I think is more readable on what is happening.

Given that its a release time only command, it doesn't need to be configurable.

WDYT?